### PR TITLE
[CPU] [ARM64] jit exp

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *[CPU] [AARCH64] jit sigmoid*

### Tickets:
 - *CVS-133071*

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@xuchen-intel Please review the PR

**Review by xuchen-intel — CHANGES_REQUESTED** [review]

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp [hunk 1]
```diff
@@ -72,6 +72,136 @@ std::set<std::vector<element::Type>> jit_add_emitter::get_supported_precisions(c
...
+
+size_t jit_exp_emitter::get_aux_vecs_count() const { return 4; }
+
+size_t jit_exp_emitter::get_aux_gprs_count() const { return 1; }
+
+void jit_exp_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel");
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: done, thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp [hunk 2]
```diff
@@ -72,6 +72,136 @@ std::set<std::vector<element::Type>> jit_add_emitter::get_supported_precisions(c
...
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+void jit_exp_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (exec_prc_ != ov::element::f32) {
+        OPENVINO_THROW("unsupported precision: " + exec_prc_.to_string());
+    }
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: done thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_emitter.cpp [hunk 3]
```diff
@@ -117,12 +117,39 @@ void jit_emitter::emitter_preamble(const std::vector<size_t>& in_idxs,
         h->str(Xbyak_aarch64::XReg(preserved_gpr_idxs[i]), pre_ptr(h->sp, -16));
     }
 
+    const size_t aux_vec_idxs_size = aux_vec_idxs.size();
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_emitter.cpp]: removed, fix will be in separated PR: https://github.com/openvinotoolkit/openvino/pull/23387 (wip)

**Comment by dmitry-gorokhov** [comment]
@xuchen-intel Any remaining comments?

**Review by xuchen-intel — APPROVED** [review]

**Comment by xuchen-intel** [comment]
> @xuchen-intel Any remaining comments?

The other parts LGTM. Thanks Dmitry for the reminder!

**Review by dmitry-gorokhov — APPROVED** [review]

**Comment by dmitry-gorokhov** [comment]
@eshoguli Please rebase the PR

**Pull request merged**